### DISCUSSION TITLE
Calculated column only on display

### DISCUSSION THREAD

**Discussion by carolin-skiply** [directus/directus#13832] [Archive]
Hi guys,

I have 2 DateTime columns (Arrival and Departure) for my use case. I would like only on the display side calculate the difference between those 2 (for instance, if arrival = 12:00pm and departure = 2:00pm, time on site = 120 minutes). 
Is that possible (without having a dedicated column within my database) ? 
Or is the only way to do so: to have a dedicated column that calculate it when writing the departure time ?

Thanks for your help ?

**Comment by rijkvanzanten** [comment]
There's currently no calculated fields like that. Keep an eye on https://github.com/directus/directus/discussions/3079 for that 🙂

**Reply by carolin-skiply** [reply]
Thanks
